docs/one-click-integrations/github-integration.md

```diff
@@ -77,7 +77,7 @@ The following is a detailed description of how the Pulse GitHub integration auto
 
 -   In this case, you must send to Pulse the information about your **deployments** and the corresponding **changes** using the [Pulse CLI](../cli/cli.md) or the [Ingestion API](https://ingestion.pulse.codacy.com/v1/api-docs).
 
-## Collected data
+## Collected data {: id="collected-data"}
```

Comment:
Adding this explicit ID isn't necessary because MkDocs already generates the same ID for this section title automatically. You can check that here:

https://docs.pulse.codacy.com/one-click-integrations/github-integration/#collected-data

We usually define these explicit IDs only when the section titles are very long or are prone to change (to avoid breaking incoming links).

```suggestion
## Collected data
```